std/shared/src/main/scala/cats/effect/std/Dispatcher.scala

```diff
@@ -25,6 +25,8 @@ import scala.collection.immutable.LongMap
 import scala.concurrent.{Future, Promise}
 
 import java.util.concurrent.atomic.{AtomicBoolean, AtomicReference}
+import scala.util.Success
+import scala.util.Failure
```

Comment:
Random stupid preference:

```suggestion
import scala.util.{Failure, Success}
```